structs.wdl

```diff
@@ -0,0 +1,34 @@
+version 1.0
+
+import "tasks/bwa.wdl" as bwa
+import "tasks/common.wdl" as common
+
+struct Readgroup {
+    String id
+    File R1
+    String R1_md5
```

Comment:
I think it's more user friendly to make this option optional, same I did for the RNA pipeline